ground_truth_labeling_jobs/3d_dense_point_cloud_downsampling_tutorial/ground_truth_annotation_dense_point_cloud_tutorial.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>Add description above:</p><p> </p><p>"In the following cell you configure the labeling job request. To learn more about the parameters used to configure a labeling job, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-create-labeling-job-api.html" target="_blank">Create a Labeling Job (API)</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Updated it</p>